packages/account-v2/src/pages/ConnectedApps/ConnectedAppsTable.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { OauthApps } from '@deriv/api-types';
+import { Button, Table } from '@deriv-com/ui';
+import { connectedAppsHeader, connectedAppsHeaderOrder, getConnectedAppsScopes } from '../../utils/connectedAppsUtils';
+
+type TConnectedAppsTable = {
+    connectedApps: OauthApps;
+    // handleToggleModal: (app_id: number) => void;
```

Comment:
Remove the commented code.

Reply:
Kept it for future reference 😄 , but I'll remove it 🙏